app/templates/main.php

```diff
@@ -0,0 +1,65 @@
+<section class="content__side">
+    <h2 class="content__side-heading">Проекты</h2>
+
+    <nav class="main-navigation">
+        <ul class="main-navigation__list">
+            <?php foreach ($projects as $project): ?>
+                <li class="main-navigation__list-item">
+                    <a class="main-navigation__list-item-link" href="#"><?= htmlspecialchars($project); ?></a>
+                    <span class="main-navigation__list-item-count"><?= tasksСounter($tasks, $project); ?></span>
+                </li>
+            <?php endforeach; ?>
+        </ul>
+    </nav>
+
+    <a class="button button--transparent button--plus content__side-button"
+       href="pages/form-project.html" target="project_add">Добавить проект</a>
+</section>
+
+<main class="content__main">
+    <h2 class="content__main-heading">Список задач</h2>
+
+    <form class="search-form" action="index.php" method="post" autocomplete="off">
+        <input class="search-form__input" type="text" name="" value="" placeholder="Поиск по задачам">
+
+        <input class="search-form__submit" type="submit" name="" value="Искать">
+    </form>
+
+    <div class="tasks-controls">
+        <nav class="tasks-switch">
+            <a href="/" class="tasks-switch__item tasks-switch__item--active">Все задачи</a>
+            <a href="/" class="tasks-switch__item">Повестка дня</a>
+            <a href="/" class="tasks-switch__item">Завтра</a>
+            <a href="/" class="tasks-switch__item">Просроченные</a>
+        </nav>
+
+        <label class="checkbox">
+            <!--добавить сюда атрибут "checked", если переменная $show_complete_tasks равна единице-->
+            <input class="checkbox__input visually-hidden show_completed"
+                   type="checkbox"
+                   <?php if ($show_complete_tasks === 1): ?>checked<?php endif; ?>
```

Comment:
Можно проще, смотри:
```suggestion
                   <?= $show_complete_tasks === 1 ? 'checked' : '' ?>
```

Reply:
`<?=` короткий echo
ну а тернарный оператор должен быть тебе знаком

Reply:
Ну да, так читаемость повышается, меня что-то перемкнуло, что тернарнике, кода мы для false передаём пустую строку - это не очень красиво. Спасибо.